dop.sh

```diff
@@ -0,0 +1,18 @@
+## Install helm
```

Comment:
Please add linting check to verify the correctness of this script.
Refer shellcheck!
- https://github.com/koalaman/shellcheck